tutorials/migration.md

```diff
@@ -464,6 +464,8 @@ cp <openHAB 1.x conf>/configurations/sitemaps/* <openHAB 2 conf>/sitemaps/*
   * Static webview files are now located in `<openhab 2 conf>/html` instead of  `/usr/share/openhab/webapps`
 
   * The name of the sitemap (i.e. the word right after `sitemap` at the top of the file) must match the file name. For example the file named `myhome.sitemap` should start with `sitemap myhome`.
+  * Charts no longer display a legend by default. If this parameter is not set, the legend is hidden if there is only one chart series.
```

Comment:
Please change here to one line per sentence.
It will not affect the later look of the page.

We do this to keep track of changes on github better.